src/runtime/utils/url.ts

```diff
@@ -10,6 +10,7 @@ const getApiURL = () => {
 
 export const getRequestURL = (includePath = true) => _getURL(useRequestEvent()?.node.req, includePath)
 export const joinPathToApiURL = (path: string) => joinURL(getApiURL(), path)
+export const publicRuntimeConfig = () => useRuntimeConfig().public
```

Comment:
please remove this and directly access it instead:
- there's some caveats with using nested composables in Nuxt 3 that we want to avoid,
- we'd need to change more places that access the runtime config to make use of this

Reply:
the reason for me to put this here was the comment here
https://github.com/sidebase/nuxt-auth/blob/d303114130cb8b84d3f7afe5876d593939dd64c3/src/runtime/composables/useSession.ts#L78


<img width="150px" src="https://media.tenor.com/mZpwbjGSt_IAAAAC/cathy-newman-newman.gif"/>

... that i should just use the `useRuntimeConfig()` within the `useSession.ts` ?

if that is the case shouldn't we then also rework the lines that have the workaround for this?